### DISCUSSION TITLE
Help Request: Unable to execute gradelew.bat

### DISCUSSION THREAD

**Discussion by rajatmohan22** [TEAMMATES/teammates#12554] [Help requests] [answered]
### Environment

- **Commit SHA: Result of trying to execute gradelew.bat
- **Operating System: Windows 11
- **JDK Version:11
- **Node.js Version:v18.17.1

### Intention

I am trying to build teammates on my local system and when i try to run gradelew.bat, I get the error saying: 

```
* What went wrong:
A problem occurred configuring root project 'teammates'.
> Could not open cp_proj generic class cache for build file 'C:\Users\lkraj\Documents\Projects\teammates\build.gradle' (C:\Users\lkraj\.gradle\caches\7.5\scripts\e6xmf8avaw1no3toif9ut9cj7).
   > BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 64

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
```

Please help me out! I am using jdk 11 as recommended 

### Resolution Attempt

I got the jdk 11 as mentioned in docs (previously had jdk 20)

**Comment by wkurniawan07** [comment, marked as answer]
> BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' **Unsupported class file major version 64**

Unfortunately this means you're using Java 20.
